src/templates/haml/get_started/glossary.html.haml

```diff
@@ -7,8 +7,9 @@
     %p= l('The barrier of a binary option trade is the price target you set for the underlying. You can choose trades that stay below or go above a price target, or stay between two targets.')
     %h2= l('Binary option')
     %p= l('A binary option is a contract purchased by a trader, which pays a pre-determined amount if their prediction is correct.')
-    %h2= l('Commodities')
-    %p= l('Commodities are resources that are grown or extracted from the ground, such as silver, gold and oil. On [_1], they are priced in US dollars.', stash->{website_name})
+    .ja-hide
+      %h2= l('Commodities')
```

Comment:
we could have used it like %h2.ja-hide, with current implementation it will create a div container like `<div class='ja-hide'><h2>Commodities</h2>...</div>`